### ISSUE TITLE
[Recipe Request] Discourse

### ISSUE THREAD

**Issue by keunes** [getferdi/recipes#195]
Not familiar with Javascript, but would like to request support for the following service:

Name of the service (e.g. WhatsApp): Discourse
URL of the service (e.g. whatsapp.com): https://www.discourse.org/

It was requested also in Franz's repo: https://github.com/meetfranz/franz/issues/1278

GitHub URL to a recipe : https://github.com/chr92/franz-discourse

**Comment by kkplein** [comment]
Yes! We were surprised to see that discourse is not yet included in ferdi!

**Comment by Oshyan** [comment]
I too would really like to see Discourse support!

**Comment by LBCreative** [comment]
It would be great if Ferdi has Discourse support! I really enjoy the app, but this is a missed opportunity at the moment.
